sourcemash/frontend/frontend.py

```diff
@@ -1,9 +1,13 @@
 from . import bp
 
-from flask import render_template
+from flask import render_template, url_for
 
 
 @bp.route('/')
 @bp.route('/index')
 def index():
     return render_template('index.html')
+
+@bp.route('/login')
```

Comment:
I'm playing around with the branch locally and I don't think it's these lines that are causing either 1 or 2. The template is set in the default config file and the url it's hitting is in flask-security. Still can't figure out why the submit isn't working though.